src/classes/DoubleLinkedList.js

```diff
@@ -0,0 +1,77 @@
+class DoubleLinkedList {
+    constructor(){
+        this.first = null;
+        this.last = null;
+        this.len = 0;
+    }
+
+    push(node) {
+        this.len += 1;
+        if (this.first === null) {
+            this.first = node;
+            this.last = node;
+        } else {
+            this.last.next = node;
+            node.prev = this.last;
+            this.last = node;
+        }
+    }
```

Comment:
This is interesting. Usually, this kind of method receives a value, and the linked list creates the node.

Your approach is interesting because you are applying dependency isolation, so the method receives an instance of an object which should have a prev property setter (in this case) that allows another instance of a node.

It's okay, but in theory, nodes are part of linked lists, so you don't have to isolate the dependency. It can be a strong dependency, and the method should receive a value, which will go into the linked list node.

Reply:
Yes, it's a better approach having that type of relation